spec.bs

```diff
@@ -1071,35 +1090,49 @@ nested scope), renumbering this and later steps as necessary.
 Issue: Need to handle `auctionReportBuyers` and `auctionReportBuyerKeys` here or
     in the Protected Audience API spec.
 
-The <a spec="turtledove">evaluate a script</a> steps are modified in two ways.
-First, the following steps are added after step 11 ("If
+The <a spec="turtledove">evaluate a script</a> steps are modified in three ways.
+First, we add two new parameters: an <a spec="turtledove">auction config</a>
+|auctionConfig| and an [=interest group=] or null |ig|.
+
+Second, we add the following steps after step 11 ("If
 <var ignore>evaluationStatus</var> is an [=ECMAScript/abrupt completion=]..."),
 renumbering later steps as appropriate:
-<div algorithm="protected-audience-evaluate-a-script-monkey-patch-1">
-12. Let |debugScope| be a new [=debug scope=].
-1. Set <var ignore>global</var>'s {{InterestGroupScriptRunnerGlobalScope/
+<div algorithm="protected-audience-evaluate-a-script-monkey-patch">
+12. If |auctionConfig|'s [=auction config/permissions policy state=]'s
```

Comment:
Similar to shared storage, probably can just set the field without checking the value, and early return?

Reply:
Switched the setting, but left early return as above